src/components/Book.js

```diff
@@ -3,16 +3,41 @@ import React from 'react';
 const Book = (props) => {
     const sessions = props.book.sessions
     const timestamp = sessions[sessions.length - 1].timestamp
-    // console.log(timestamp)
     const dateCompleted = new Date(timestamp)
-    // console.log(dateCompleted)
+
+    // go through the sessions array for each book and find the total time spent reading the book
```

Comment:
You could define what wholeHours, wholeMinutes and wholeSeconds are here and remove all of the comments following. There is a balance between the right amount of comments and over commenting and there is no right answer. But having well named constants/variables, which you have, goes a long way to reduce the comments you need. You could even remove all of these comments entirely because the variables are so well named, or alternatively just have this alone here.

"Find the total time spent reading by whole units."